### PR TITLE
To do list

### PR DESCRIPTION
**Pull request by Kajal19-del**
# In this milestone I did the following activities:
- I setup webpack first
- I add html, css and javascript files for to-do-list project
- I pass  all the linter errors

### CODE REVIEW

**Comment by julie-ify** [comment]
Hi @Kajal19-del,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### Kindly add the table of contents from the readme template and ensure the links work.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,167 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](N/A)
+
+## 🚀 Live Video <a name="live-video"></a>
+
+- [Live Video Link](N/A)
+
```
Comment (ABDULALI3468) [inline README.md]: As you have not deployed the project so it's showing a 404 error. If you want to deploy you can do it with GitHub pages or simply remove this section in those projects where you don't deploy the project online.

**Review by ABDULALI3468 — COMMENTED** [review]
Hi @Kajal19-del  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request


**_Check the comment below to understand the reason for invalidation!_**



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (Kajal19-del) [reply README.md]: Thank You

**Review by julie-ify — APPROVED** [review]
Hi @Kajal19-del,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**